code_annotations/base.py

```diff
@@ -578,66 +578,33 @@ def _format_results_for_report(self, all_results):
         Returns:
             Dict of results arranged for reporting
         """
-        group_children = self._get_group_children()
         formatted_results = {}
         current_group_id = 0
-
         for filename in all_results:
-            self.echo.echo_vv(f"report_format: formatting {filename}")
             formatted_results[filename] = []
-            current_group = None
-
-            found_group_members = []
-
-            for annotation in all_results[filename]:
-                token = annotation['annotation_token']
-                self.echo.echo_vvv(f"report_format: formatting annotation token {token}")
-
-                if current_group:
-                    if token not in self.config.groups[current_group]:
-                        self.echo.echo_vv(
-                            "report_format: {} is not a group member, finishing group id {}".format(
```

Comment:
Is `iter_groups` a breaking change from this? Do we now fail, or is it the same thing?

Reply:
See my comment below, in the test module.